### DISCUSSION TITLE
Many things are easier; nothing is harder

### DISCUSSION THREAD

**Discussion by tobyhodges** [carpentries/workbench#23] [Testimonials]
The Workbench templates and `{sandpaper}`'s `create_lesson` and `create_episode` functions have made it easier to start writing a new lesson; the simplified directory structure has made it easier for newcomers to contribute to a lesson; the reduced and more universal set of dependencies have made it easier to build lessons locally, which makes authoring, contributing to, and maintaining a lesson much less frustrating (it has been very common for lesson developers/Maintainers to simply give up on the idea of previewing lesson builds on their local system with the previous template); the automated checks make it easier to make sure lesson content is accessible; the fenced div syntax makes it easier to write exercises and other formatted chunks of lesson content; the internal link syntax makes it easier to link correctly between pages in a lesson (another common stumbling point with the previous template); and the _Instructor View_ and inline Instructor Notes display has made it easier to teach lessons in a workshop.

In short, the Workbench has made it easier for people to write, maintain, and teach lessons, in many different ways - sometimes quite profoundly - and so far I have not encountered anything that has been made more difficult.

\* _(The above list is non-exhaustive e.g. I didn't mention the simplified commit history for lesson projects, easier process for updating to the latest version of the template, etc...)_
